anitya/lib/backends/maven.py

```diff
@@ -53,9 +58,18 @@ def get_versions(cls, project):
             when the versions cannot be retrieved correctly
 
         '''
-        url = 'http://repo1.maven.org/maven2/{path}'\
-              .format(path=project.name.replace('.', '/').replace(':', '/'))
-
-        regex = r'\<a[^>]+\>(\d[^</]*)'
-
-        return get_versions_by_regex(url, regex, project)
+        if re.match(r'https?://repo\d+.maven.org/', project.homepage):
+            url = project.homepage
+            coordinates = None
+        else:
+            coordinates = project.version_url or project.name
+        if coordinates:
```

Comment:
We could drop this ``if`` here and put the entire code block in the ``else`` above, no?